src/test/api/prompt.test.ts

```diff
@@ -0,0 +1,23 @@
+import { deepStrictEqual, ok, strictEqual } from "node:assert";
+import test from "node:test";
+import { promptMock } from "../../mock/api/prompt";
+
+test("prompt", async (t) => {
+  const baseApi = Symbol("Base API");
+
+  await t.test("render", async (t) => {
+    const { build, state } = promptMock("Another Flow");
+    const api = build(baseApi);
+
+    strictEqual(state.rendered.length, 0);
```

Comment:
Just so I understand here: is it possible to render more than one prompt? 🤔

Reply:
I just tested and you're right, only one form can be rendered.
I have updated the state to reflect this :)